src/app/static/src/app/types.ts

```diff
@@ -69,7 +69,7 @@ export interface Snapshot {
     updated: string
 }
 
-export interface Version {
```

Comment:
On line 84 below, should that be versionId, with corresponding changes in a number of places:

```
src/app/static/src/app/types.ts
84:    versionId: number,

src/app/static/src/app/components/projects/ProjectHistory.vue
50:        loadSnapshot: (event: Event, versionId: number, snapshotId: string) => void,
64:           loadSnapshot(event: Event, versionId: number, snapshotId: string) {
66:               this.loadAction({versionId, snapshotId});

src/app/static/src/app/store/projects/actions.ts
76:            .get<SnapshotDetails>(`project/${snapshot.versionId}/snapshot/${snapshot.snapshotId}`)

src/app/static/src/tests/integration/projects.itest.ts
87:        const versionId = createdProject.id;
90:            actions.loadSnapshot({commit, dispatch, state, rootState} as any, {versionId, snapshotId});

src/app/static/src/tests/projects/actions.test.ts
286:        actions.loadSnapshot({commit, dispatch, state, rootState} as any, {versionId: 1, snapshotId: "testSnapshot"});
303:        actions.loadSnapshot({commit, dispatch, state, rootState} as any, {versionId: 1, snapshotId: "testSnapshot"});

src/app/static/src/tests/components/projects/projectHistory.test.ts
139:        expect(mockLoad.mock.calls[0][1]).toStrictEqual({versionId: 1, snapshotId: "s11"});
```